velox/exec/LocalPlanner.cpp

```diff
@@ -321,6 +324,17 @@ std::shared_ptr<Driver> DriverFactory::createDriver(
                 planNode)) {
       operators.push_back(
           std::make_unique<EnforceSingleRow>(id, ctx.get(), enforceSingleRow));
+    } else if (
+        auto assignUniqueId =
+            std::dynamic_pointer_cast<const core::AssignUniqueId>(planNode)) {
+      auto uniqueTaskId =
+          (int32_t)std::hash<std::string>{}(ctx->task->taskId());
```

Comment:
We discussed that we cannot use hash because there can be collisions. Instead, AssignUniqueId plan node should require a 32-bit integer that uniquely identifies the task to be provided by the planner.